packages/esbuild-plugin/README.md

```diff
@@ -0,0 +1,114 @@
+# Honeybadger's esbuild Source Map Plugin
+
+[esbuild](https://esbuild.github.io/) plugin to upload JavaScript
+sourcemaps and optionally send deployment notifications to [Honeybadger](https://docs.honeybadger.io/lib/javascript/guides/using-source-maps/).
```

Comment:
```suggestion
source maps and optionally send deployment notifications to [Honeybadger](https://docs.honeybadger.io/lib/javascript/guides/using-source-maps/).
```

Reply:
there are a lot of these, so maybe you can do a global find/replace.

- "sourcemap" should be "source map"
- "Sourcemap" should be "Source map"

Reply:
Thanks! I will apply this in all the monorepo.